docs/source/developers/python.rst

```diff
@@ -287,6 +292,8 @@ to ``OFF``:
 * ``ARROW_ORC``: Support for Apache ORC file format
 * ``ARROW_PARQUET``: Support for Apache Parquet file format
 * ``ARROW_PLASMA``: Shared memory object store
+* ``ARROW_WITH_{ZLIB, ZSTD, LZ4, SNAPPY, BROTLI}``: Build support for
```

Comment:
And BZ2?

Reply:
Not needed for Parquet but I'm adding here anyhow